EIPS/eip-7656.md

```diff
@@ -0,0 +1,535 @@
+---
+eip: 7656
```

Comment:
```suggestion
eip: 7513
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Thank you so much for your help, and I'm sorry for not noticing this earlier. I'd like to ask if there's any documentation regarding EIP numbering rules? Is it documented in EIP-1?